packages/json/src/parser/entry.ts

```diff
@@ -44,4 +44,5 @@ export function json(dumpErrors = false): core.Parser<JsonNode> {
 	}
 }
 
-export const entry: core.Parser<JsonNode> = json(true)
+export const entry = jsonParser(true)
```

Comment:
i removed the explicit typing here since it was already receiving the same type from `json(true)` anyway